docs/connect-blockchain/multi-chain.mdx

```diff
@@ -0,0 +1,378 @@
+          rpcTarget: "https://rpc.ankr.com/avalanche-c",
+          // Avoid using public rpcTarget in production.
+          // Use services like Infura, Quicknode etc
+          displayName: "Avalanche C-Chain Mainnet",
+          blockExplorer: "https://subnets.avax.network/c-chain",
+          ticker: "AVAX",
+          tickerName: "AVAX",
+        },
+      },
+    });
+await avaxPrivateKeyProvider.setupProvider(privateKey);
+const web3 = new Web3(avaxPrivateKeyProvider.provider as any);
+const address = (await web3.eth.getAccounts())[0];
+
+```
+
+</TabItem>
+<TabItem value="cronos">
+
+```js
+import { EthereumPrivateKeyProvider } from "@web3auth/ethereum-provider";
+
+/*
+  Use code from the above Initializing Provider here
+*/
+
+const privateKey = provider.request({ method: "eth_privateKey" });
+const cronosPrivateKeyProvider = new EthereumPrivateKeyProvider({
+      config: {
+        chainConfig: {
+          chainId: "0x19",
+          rpcTarget: "https://rpc.cronos.org",
+          // Avoid using public rpcTarget in production.
+          // Use services like Infura, Quicknode etc
+          displayName: "Cronos Mainnet",
+          blockExplorer: "https://cronoscan.com/",
+          ticker: "CRO",
+          tickerName: "CRO",
+        },
+      },
+    });
+await cronosPrivateKeyProvider.setupProvider(privateKey);
+const web3 = new Web3(cronosPrivateKeyProvider.provider as any);
+const address = (await web3.eth.getAccounts())[0];
+
+```
+
+</TabItem>
+<TabItem value="evm">
+
+```js
+import { EthereumPrivateKeyProvider } from "@web3auth/ethereum-provider";
+
+/*
+  Use code from the above Initializing Provider here
+*/
+
+const privateKey = provider.request({ method: "eth_privateKey" });
+const PrivateKeyProvider = new EthereumPrivateKeyProvider({
+      config: {
+        chainConfig: {
+          chainId: "YOUR_CHAIN_ID",
+          rpcTarget: "YOUR_RPC_URL",
+          // Avoid using public rpcTarget in production.
+          // Use services like Infura, Quicknode etc
+          displayName: "YOUR_CHAIN_NAME",
+          blockExplorer: "YOUR_BLOCK_EXPLORER_URL",
+          ticker: "YOUR_TICKER", // eg: MATIC, BNB, AVAX, CRO
+          tickerName: "YOUR_TICKER_NAME", // eg: MATIC, BNB, AVAX, CRO
+        },
+      },
+    });
+await PrivateKeyProvider.setupProvider(privateKey);
+const web3 = new Web3(PrivateKeyProvider.provider as any);
+const address = (await web3.eth.getAccounts())[0];
+
+```
+
+</TabItem>
+
+</Tabs>
+
+### Solana
+
+To interact with the Solana blockchain, you can use [@solana/web3.js](https://solana-labs.github.io/solana-web3.js/) library with Web3Auth along with
+`@web3auth/solana-provider` package.
+
+```js
+import {
+  SolanaPrivateKeyProvider,
+  SolanaWallet,
+} from "@web3auth/solana-provider";
+
+/*
+  Use code from the above Initializing Provider here
+*/
+
+const privateKey = provider.request({ method: "eth_privateKey" });
+const { getED25519Key } = await import("@toruslabs/openlogin-ed25519");
```

Comment:
@Saviour1001 please change this to use the function from `@web3auth/solana-provider` package.